client/actions/agenda.js

```diff
@@ -92,8 +92,14 @@ const selectAgenda = (agendaId, params = {}) => (
 
         // update the url (deep linking)
         $timeout(() => ($location.search('agenda', agendaId)));
+
         // reload the plannings list
-        return dispatch(fetchSelectedAgendaPlannings(params));
+        dispatch(main.setUnsetUserInitiatedSearch(true));
+        return dispatch(fetchSelectedAgendaPlannings(params))
+            .then((data) => {
+                dispatch(main.setUnsetUserInitiatedSearch(false));
```

Comment:
Dispatch this action in the `finally` phase of a `Promise`, to ensure the value is reset in all conditions (similar to how `setUnsetLoadingIndicator` uses `finally`).